lib/presentation/pages/top/top_page.dart

```diff
@@ -1,65 +1,65 @@
+import 'package:flutter/cupertino.dart';
 import 'package:flutter_riverpod/flutter_riverpod.dart';
 
+// TODO 全体的にCupertinoDesignを適用すること
 class TopPage extends ConsumerWidget {
   const TopPage({super.key});
 
   @override
   Widget build(BuildContext context, WidgetRef ref) {
-    return Scaffold(
-      appBar: AppBar(
-        title: const Align(
-          alignment: Alignment.centerLeft,
-          child: Padding(
-            padding: EdgeInsets.only(left: 16),
-            child: Text(
-              'Habit 365',
-              style: TextStyle(
-                fontWeight: FontWeight.bold,
-                fontSize: 30,
-              ),
-            ),
-          ),
-        ),
-        actions: [
-          IconButton(
-            icon: const Icon(Icons.settings),
-            onPressed: () => Navigator.of(context).push(SettingsPage.route()),
+    return CupertinoPageScaffold(
+      // backgroundColor: ,
+      navigationBar: const CupertinoNavigationBar(
+        leading: Icon(CupertinoIcons.add),
+        trailing: Icon(CupertinoIcons.settings),
+        middle: Text(
+          'Habit365',
+          style: TextStyle(
+            fontSize: 20,
+            fontWeight: FontWeight.bold,
           ),
-          const SizedBox(width: 16),
-        ],
-      ),
-      body: SafeArea(
-        child: PageView.builder(
-          controller: PageController(viewportFraction: 0.9),
-          itemCount: 4,
-          itemBuilder: (context, index) {
-            return Column(
-              children: [
-                const SummaryCard(),
-                const SizedBox(height: 16),
-                MiddleNavigationBar(index: index),
-                Expanded(
-                  child: MiddleNavigationItems(index),
-                ),
-              ],
-            );
-          },
         ),
       ),
-      floatingActionButton: FloatingActionButton(
-        child: const Icon(Icons.add),
-        onPressed: () {},
-      ),
-      // TODO: set Advertising
-      bottomNavigationBar: const ColoredBox(
-        color: Colors.amberAccent,
-        child: SizedBox(
-          height: 80,
+      child: SafeArea(
+        child: Column(
+          children: [
+            Expanded(
+              child: PageView.builder(
+                controller: PageController(viewportFraction: 0.9),
+                itemCount: 4,
+                itemBuilder: (context, index) {
+                  return Column(
+                    children: [
+                      const SizedBox(height: 32),
+                      const SummaryCard(),
+                      Expanded(
+                        child: MiddleNavigationItems(index),
+                      ),
+                      Row(
+                        mainAxisAlignment: MainAxisAlignment.center,
+                        children: [
+                          CupertinoButton.filled(
+                            child: Icon(CupertinoIcons.pen),
```

Comment:
<table data-meta="generated_by_danger-pr">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/kolbe-ryo/count_habits/blob/ac49fafa1049debe629d827640372662ab94aa23/lib/presentation/pages/top/top_page.dart#L45"></span>Use 'const' with the constructor to improve performance</td>
    </tr>
  </tbody>
</table>